daml-lf/interpreter/src/test/scala/com/daml/SBuiltinInterfaceTest.scala

```diff
@@ -347,17 +347,19 @@ object SBuiltinInterfaceTest {
       getContract: PartialFunction[Value.ContractId, Value.VersionedContractInstance],
       getKey: PartialFunction[GlobalKeyWithMaintainers, Value.ContractId],
   ): Try[Either[SError, SValue]] = {
-    val machine =
-      if (onLedger)
+    if (onLedger) {
```

Comment:
We should the usage of this function to be use only in "on-ledger". 
I suspect we used only once in "off-ledger" mode, where we can directly use  
`Speedy.Machine.fromPureSExpr(compiledBasePkgs, e).runPure()`

Reply:
We should be able to kill `SpeedyTestLib.runPure`